eng/scripts/Create-APIView.ps1

```diff
@@ -111,15 +117,17 @@ function Invoke-SwaggerAPIViewParser {
       LogGroupEnd
 
       $generatedAPIViewTokenFile = Get-ChildItem -File | Select-Object -First 1
```

Comment:
Do we not know the expected outputted file name?

Reply:
Is the file named after the tag? If so is that a hacky way of getting the tag name as output or does it make since for it to be named that and we should get that value another way?

Reply:
Yes the output file is name is known. The output file names if the name of the tag used to create the token file.

Reply:
Don't we know the name of the file now? Why are we still calling Get-ChildItem here?